airbyte-integrations/connectors/source-relational-db/src/main/java/io/airbyte/integrations/source/relationaldb/DbSourceDiscoverUtil.java

```diff
@@ -50,14 +59,28 @@ public static <DataType> void logSourceSchemaChange(final Map<String, TableInfo<
           .stream()
           .map(commonField -> toField(commonField, airbyteTypeConverter))
           .distinct()
-          .collect(Collectors.toList());
+          .collect(toList());
       final JsonNode currentJsonSchema = fieldsToJsonSchema(fields);
-
       final JsonNode catalogSchema = stream.getJsonSchema();
-      if (!catalogSchema.equals(currentJsonSchema)) {
+      final JsonNode currentSchemaProperties = currentJsonSchema.get("properties");
+      final JsonNode catalogProperties = catalogSchema.get("properties");
+      final List<String> mismatchedFields = new ArrayList<>();
+      catalogProperties.fieldNames().forEachRemaining(fieldName -> {
+        if (AIRBYTE_METADATA.contains(fieldName)) {
```

Comment:
Just add a quick comment on what we're doing here (why we're ignoring the CDC metadata fileds)

Reply:
+1. On it.